docs/content/users/install/ddev-installation.md

```diff
@@ -137,11 +137,11 @@ Once you’ve [installed a Docker provider](docker-installation.md), you’re re
 
     You can install DDEV on Windows three ways:
 
-    1. [Using WSL2 with Docker inside](#wsl2-docker-ce-inside-install-script)
-    2. [Using WSL2 with Docker Desktop](#wsl2-docker-desktop-install-script)
-    3. [Installing directly on traditional Windows](#traditional-windows) with an installer
+    1. **Most reliable, free, open-source:** [Using WSL2 with Docker inside](#wsl2-docker-ce-inside-install-script)
+    2. **Commercial, often problematic:** [Using WSL2 with Docker Desktop](#wsl2-docker-desktop-install-script)
+    3. **Old school:** [Installing directly on traditional Windows](#traditional-windows) with an installer
```

Comment:
```suggestion
    3. **Old school:** [Installing without WSL2 on traditional Windows](#traditional-windows) with an installer
 ```

Reply:
I wonder how to say this better.  This goes back to the same confusion about WSL2. You can't install Docker Desktop (required for old-school Windows setup) without WSL2, because that's what it uses under the hood. But with this solution, **DDEV** is used from git-bash or PowerShell, and it uses Docker Desktop.